app/presenters/quadicons/quadrants/host_vendor.rb

```diff
@@ -0,0 +1,14 @@
+module Quadicons
+  module Quadrants
+    class HostVendor < Quadrants::Base
+      def path
+        "svg/vendor-#{vendor}.svg"
```

Comment:
same here .. also, this looks suspiciously like what `record.decorate.fileicon` does..